### PR TITLE
Changes during Norland Hub 2 provisioning (CU-vf96g1)

### PR DESCRIPTION
**Pull request by trisapeace**
Some updates to the README and to `setup_pi.yaml` as we were going through the provisioning process for Norland Hub 2.

## Test plan
- :heavy_check_mark: Provision Norland Hub 2

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-vf96g1 review and finalize documentation for Ansible and the config repo](https://app.clickup.com/t/vf96g1)

File: README.md [hunk 1]
```diff
@@ -69,7 +69,13 @@
...
-1. follow the generic RPi setup instructions above
+1. use either Balena Etcher or Raspberry Pi Imager to flash the SD card with Raspbian Buster
+
+1. create a file named `ssh` in the boot partition of the SD card
+
+1. install the SD card into the raspberry pi, connect it to power and ethernet
+
+1. plug the RPi into a router that you have admin access to and determine its IP address
 
 1. install ansible (see https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
```
Comment (schwarrrtz) [inline README.md:80]: would it make sense to change this to 'on your local machine, install ansible ...' to clarify that the instructions no longer refer to the RPi environment?

Comment (trisapeace) [reply README.md]: Good idea. Done.

**Review by schwarrrtz — APPROVED** [review]
looks good, thanks for tracking these changes during our provisioning demo session.

**Pull request merged**